source/Core/BSP/Miniware/configuration.h

```diff
@@ -247,18 +249,14 @@
 #define TEMP_NTC
 #define I2C_SOFT_BUS_2 1
 #define SC7_ORI_FLIP
-#endif
-#endif
-
-#ifdef MODEL_TS101
-#define FLASH_LOGOADDR (0x08000000 + (126 * 1024))
-
-#else
-#define FLASH_LOGOADDR (0x08000000 + (62 * 1024))
-#endif
+#endif /* TS80P */
 
 #ifdef MODEL_TS101
```

Comment:
Wouldn't you find this way more readable? If no, I will revert it without a question.

Reply:
Yeah this is a fine cleanup